src/components/Chart/ChartOptions.jsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React, { useEffect, useState } from 'react';
+import Dropdown from 'react-bootstrap/Dropdown';
+import Form from 'react-bootstrap/Form';
+import SelectableContext from 'react-bootstrap/SelectableContext';
+import { useDispatch, useSelector } from 'react-redux';
+import { NumberInlineInput } from 'pc-nrfconnect-shared';
+import { func, number } from 'prop-types';
+
+import { chartState, setYMax, setYMin } from '../../reducers/chartReducer';
+
+const unitLabels = ['\u00B5A', 'mA', 'A'];
+
+const ChartOptions = () => {
+    const { yMin, yMax, yAxisLock } = useSelector(chartState);
+    const dispatch = useDispatch();
+
+    const [localYMin, setLocalYMin] = useState(1);
+    const [localYMax, setLocalYMax] = useState(1);
+
+    /** Mapping to the power to get correct value of uA
+     * 0=uA, 1=mA, 2=A
```

Comment:
Perhaps these could be defined as constants? For example: 
```
const MICRO_AMP_UNIT = 0; 
const MILLI_AMP_UNIT = 1;
const AMP_UNIT = 2;
```

Not quite sure about the naming here.